library/objective-c/EnvoyTypes.h

```diff
@@ -91,6 +98,9 @@ typedef struct {
  */
 @property (nonatomic, strong) void (^onError)(EnvoyEngineError *error);
 
+// FIXME
+@property (nonatomic, strong) void (^onCancel)();
+
```

Comment:
Should we add one for `onComplete` too? Or is the assumption that `onTrailers` encompasses this?

Reply:
Current plan is not to surface onComplete, since it's actually meant more to trigger internal bookkeeping. Other callbacks have the boolean on them and trailers/error imply closure already.

That said we _can_ surface this if it ends up being necessary/useful. Currently though, it will be a little bit redundant.